### PR TITLE
Add student fee protection rule

### PR DESCRIPTION
**Pull request by NMaree**
Added new SFP rule to Student Refunds Withdrawals Fee Protection Policy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,18 @@ Development/Approval dates: October 2019<br>
            </ul></li>
          <li>4.4 Developers Institute will refund an amount equal to the sum of the amount paid.</li>
          <li>4.5 Formal withdrawal within the first eight (8) working days of the programme is confirmed by the completion and return of the Cancellation/Withdrawal Form to the Programme Coordinator. The Academic Director shall have been communicated with (by the Programme Coordinator) and signed off the form for the withdrawal to proceed. The date of withdrawal is deemed to be the date the student returns the signed Cancellation/Withdrawal form.</li>
-         <li>4.6 Students who have never attended class based on an audit of attendance, shall automatically be treated as an Enrolment Cancellation/Withdrawal with a full refund.</li>
+         <li>4.6 Students who have never attended class based on an audit of attendance, shall automatically be treated as an Enrolment Cancellation/Withdrawal with a full refund
```
Comment (fiq) [inline README.md]: Full stop missing.

**Review by fiq — APPROVED** [review]
All good, other than the missing typo. Thank you.

**Comment by NMaree** [comment]
Thank you.

File: README.md (hunk 1 above)
Comment (fiq) [reply README.md]: @NMaree since I've reviewed this, I am good with your merging the PR. Just aware that your branch is still unmerged

**Pull request merged**